elements/rh-band/src/rh-band.scss

```diff
@@ -113,96 +126,96 @@ $aside-lg: 300px;
       @media (max-width: #{rh-breakpoint(md)}) {
-        margin-bottom: 0;
-      }
-      &:last-child {
-        margin-bottom: 0;
+         margin-bottom: #{rh-var(container-spacer)};
       }
     }
   }
-  &__main > *:not(:last-child) {
-    margin-bottom: #{rh-var(container-spacer)};
-  }
+
   &__main,
   &__body {
-    @media screen and (min-width: #{rh-breakpoint(md)}) {
-      margin-right: calc(#{rh-var(container-spacer)} * 2);
-      [aside~="left"] & {
-        margin-right: 0;
-      }
-      [aside~="left"] & {
-        margin-left: calc(#{rh-var(container-spacer)} * 2);
-      }
-    }
-    @media (min-width: #{rh-breakpoint(lg)}) {
-      width: calc(100% - #{$aside-lg} + calc(#{rh-var(container-spacer)} * 2));
+    flex: 1;
+    // spacing between children:; 
+    > *:not(:last-child) {
+         margin-bottom: #{rh-var(container-spacer)};
     }
   }
+
   &__header,
   &__body,
   &__footer,
   &__aside {
     display: block;
+    display: grid;  // for elegant display of children
+    grid-column: 1fr;
+    grid-gap: #{rh-var(container-spacer)};
     &::slotted(*) {
       margin-top: 0;
     }
   }
   &__header {
-    $SLOT: header;
-    display: grid;
-    grid-template-columns: 1fr;
-    grid-gap: #{rh-var(container-spacer)};
+    $SLOT: header !global;
+    
     @media screen and (min-width: #{rh-breakpoint(md)}) {
       grid-template-columns: #{rh-local(layout)};
     }
   }
   &__body {
-    $SLOT: body;
+    $SLOT: body !global;
     flex-grow: 1;
-    display: grid;
-    grid-template-columns: 1fr;
-    grid-gap: #{rh-var(container-spacer)};
+
     @media screen and (min-width: #{rh-breakpoint(md)}) {
       grid-template-columns: #{rh-local(layout)};
-      width: calc(100% - (#{$aside-md} + calc(#{rh-var(container-spacer)} * 2)));
+
       [aside~="full"] & {
         margin-right: 0;
         margin-left: 0;
       }
     }
   }
   &__footer {
-    $SLOT: footer;
-    display: grid;
-    grid-template-columns: 1fr;
-    grid-gap: #{rh-var(container-spacer)};
+    $SLOT: footer !global;
+
     @media screen and (min-width: #{rh-breakpoint(md)}) {
       grid-template-columns: #{rh-local(layout)};
     }
   }
+
   &__aside {
-    $SLOT: aside;
-    display: grid;
-    grid-template-columns: 1fr;
-    grid-gap:  #{rh-var(container-spacer)};
-    max-width: #{$aside-md};
+    $SLOT: aside !global;
+    min-width:             $aside-sm;
+
     @media screen and (min-width: #{rh-breakpoint(md)}) {
       grid-template-columns: #{rh-local(layout)};
+      flex-basis:            #{rh-local(layout)};
     }
-    @media (min-width: #{rh-breakpoint(lg)}) {
-      width: #{$aside-lg};
-    }
-    [aside~="left"] & {
-      order: -1;
+    @media screen and (min-width: #{rh-breakpoint(md)}) and (max-width: #{rh-breakpoint(lg)}) {
+      --rh-band_aside--layout: #{$aside-sm}; // set limiter
     }
-    [aside~="right"] & {
-      order: 2;
+  }
+  &__aside {  // walk away from aside SLOT var to invoke gutter
```

Comment:
This still might need some help. The only way I could get the SLOT vars to work was to make them global, but then you have to keep overriding them

Reply:
I can help with that.  They just need to be updated before the function is run.